ext/standard/basic_functions.stub.php

```diff
@@ -960,27 +921,22 @@ function realpath_cache_size(): int {}
 
 /* formatted_print.c */
 
-/** @param mixed ...$args */
-function sprintf(string $format, ...$args): string {}
+function sprintf(string $format, mixed ...$args): string {}
 
-/** @param mixed ...$args */
-function printf(string $format, ...$args): int {}
+function printf(string $format, mixed ...$args): int {}
 
-/** @param mixed $args */
+/** @param array $args */
```

Comment:
Unrelated: We should make this an actual array argument.

Reply:
I'll take care of it then